R/aggregate_game_stats_def.R

```diff
@@ -305,11 +305,18 @@ calculate_player_stats_def <- function(pbp, weekly = FALSE) {
       values_fn = sum,
       values_fill = 0
     ) %>%
-    dplyr::filter(!is.na(.data$player_id)) %>%
+    # Renaming fails if the columns don't exist. So we row bind a dummy tibble
+    # including the relevat columns. The row will be filtered after renaming
+    dplyr::bind_rows(
+      tibble::tibble(
+        player_id = NA_character_, fumbled = 0, fumble_recovery = 0
```

Comment:
what about using numeric(0) ?

Reply:
irrelevant because that row get's filtered out 4 lines later

Reply:
or do you mean from a memory allocation perspective?

Reply:
also reduces need to filter since it just creates the column and doesn’t add the row?

Reply:
the filter was in there anyway. We can't remove it. But I will make them numeric(0)